lib/stat_tracker.rb

```diff
@@ -138,4 +147,95 @@ def percentage_visitor_wins
     end
     (games_won_by_visitor.count.to_f / games.count * 100).round(2)
   end
+
+  def count_of_teams
+    @teams.count
+  end
+
+  def games_by_all_team_ids
+    @game_teams.group_by do |game_team|
```

Comment:
@aprildagonese are the values in this hash just one of the rows of the game_team cvs, as a game_team instance? I'm kind of confused by the game team cvs thing